VERSIONING_POLICY.md

```diff
@@ -0,0 +1,33 @@
+# Open OnDemand Versioning Policy
+## Version Numbering
+Open OnDemand code releases utilize a XXX.YYY.ZZZ numbering scheme (e.g. 2.0.29), where:
+* XXX is the major version number
+* YYY is the minor version number
+* ZZZ is the patch version number
+
+## Major Versions
+New major version numbers will typically be implemented for any of the following reasons:
+* Significant breaking changes that prevent seamless upgrading from previous versions
+* Significant deprecations related to software dependencies
+* Large sets of new features or functionalities
+
+## Minor Versions
+New minor version numbers will typically be implemented for any of the following reasons:
+* New features of functionalities
+* Large sets of bug fixes
+
+## Patch Versions
+New patch version numbers will typically be implemented for any of the following reasons:
+* Security updates
+* Minor bug fixes
+
+## Tagging
+Intermediate or transient releases will be indicated via additional tagging, including:
+* Pre-release, for releases that are being actively tested in production at early adopter sites
+* rc#, for release candidate # (where # gets incremented as needed for subsequent release candidates)
+
+## Version Timing
+The Open OnDemand team anticipates releasing new major versions every 1 to 2 years.  Minor versions are generally released at most a few times per year.  Patch versions are released as needed.  All versions are typically tested first in production for at least a week at early adopter sites prior to being marked as an official release.
```

Comment:
I understand your concern about setting expectations for future releases. It's true that we might not always hit these timelines, but having a general idea of our release schedule can be helpful for several reasons:

**Project planning**: By setting targets for major, minor, and patch releases, we can allocate resources, prioritize features, and plan development work more effectively. This helps ensure that we stay focused on our goals and make consistent progress.

**Communication**: Sharing our anticipated release schedule with stakeholders, such as users, customers, and other developers, gives them an idea of what to expect from our project in terms of updates, bug fixes, and new features. This can help build trust and confidence in our ability to deliver and maintain the software.

 **Motivation**: Having a timeline can act as a motivator for the team to work towards achieving our goals. It encourages us to keep moving forward and strive for continuous improvement.

We can consider the release schedule in our VERSIONING_POLICY.md as a flexible guideline rather than a strict commitment. We could also add a note clarifying that these timelines are subject to change based on various factors, such as project requirements, resource availability, and other unforeseen circumstances.

Reply:
OK I guess. It's just a bit of a fantasy is all, so I guess we'll see 🤷‍♂️.